src/expressions/functions/builtInFunctions_node.ts

```diff
@@ -165,6 +165,78 @@ function areSameNode(
 	return true;
 }
 
+/**
+ * Test if a string should match the xml:lang attribute string of a node.
+ */
+function langMatchString(truth: string, test: string): boolean {
+	truth = truth.toLowerCase();
+	test = test.toLowerCase();
+	if (truth === test) {
+		return true;
+	} else {
+		return truth.length < 5 || !truth.startsWith(test)
```

Comment:
Are we certain no valid tags are prefixes of each other?
Also, why 5? Maybe checking for the presence of a `-` is a better exit cause for the recursion.

Reply:
The `startswith` is just a heuristic, the real testing is done after splitting off the last part.

`< 5` because it will match `en-us` and won't match `en`. Maybe it should be `truth.contains("-")`